### PR TITLE
Arch board changes for Confidential Ledger GA release

### PR DESCRIPTION
**Pull request by christothes**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by deer-prudence — APPROVED** [review]

**Comment by deer-prudence** [comment]
I believe the Java SDK currently has a [nested certificate](https://apiview.dev/Assemblies/Review/8a7672078f014c16b1173cf0665a00a6) module. Should this be changed to match dotnet?

**Comment by christothes** [comment]
> I believe the Java SDK currently has a [nested certificate](https://apiview.dev/Assemblies/Review/8a7672078f014c16b1173cf0665a00a6) module. Should this be changed to match dotnet?

I'm not sure if this is just a Java thing or not - @srnagar , could you comment?

File: sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs [hunk 1]
```diff
@@ -4,33 +4,32 @@
...
+        /// <summary> Initializes a new instance of ConfidentialLedgerCertificateClient. </summary>
+        /// <param name="certificateEndpoint"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
         /// <param name="options"> The options for configuring the client. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="identityServiceUri"/> </exception>
-        public ConfidentialLedgerIdentityServiceClient(Uri identityServiceUri, ConfidentialLedgerClientOptions options)
+        /// <exception cref="ArgumentNullException"> <paramref name="certificateEndpoint"/> </exception>
+        public ConfidentialLedgerCertificateClient(Uri certificateEndpoint, ConfidentialLedgerCertificateClientOptions options)
         {
-            if (identityServiceUri == null)
+            if (certificateEndpoint == null)
```
Comment (jsquire) [inline sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs:30]: Any reason that we're not using the Azure.Core `Argument` assertions?

File: sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/CertificateServiceSample.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```C# Snippet:GetIdentity
+Uri identityServiceEndpoint = new("https://identity.confidential-ledger.core.azure.com") // The hostname from the identityServiceUri
+var identityClient = new ConfidentialLedgerCertificateClient(identityServiceEndpoint);
+
+// Get the ledger's  TLS certificate for our ledger.
+string ledgerId = "<the ledger id>"; // ex. "my-ledger" from "https://my-ledger.eastus.cloudapp.azure.com"
+Response response = identityClient.GetLedgerIdentity(ledgerId);
+X509Certificate2 ledgerTlsCert = ConfidentialLedgerCertificateClient.ParseCertificate(response);
+
+X509Chain certificateChain = new();
```
Comment (jsquire) [inline sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/CertificateServiceSample.md:17]: This is a pretty dense block.   Maybe consider some whitespace to aid readability?

**Review by jsquire — APPROVED** [review]

File: sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Azure.Security.ConfidentialLedger.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>Client SDK for the Azure Confidential Ledger service</Description>
     <AssemblyTitle>Azure Confidential Ledger</AssemblyTitle>
-    <Version>1.0.0-beta.4</Version>
+    <Version>1.0.0</Version>
     <PackageTags>Azure ConfidentialLedger</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
```
[the remark this answers is not in the document]
Comment (christothes) [reply sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Azure.Security.ConfidentialLedger.csproj:8]: Good catch. thanks!

File: sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs (hunk 1 above)
Comment (christothes) [reply sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs:30]: nope :) Will add.

**Comment by srnagar** [comment]
> > I believe the Java SDK currently has a [nested certificate](https://apiview.dev/Assemblies/Review/8a7672078f014c16b1173cf0665a00a6) module. Should this be changed to match dotnet?
> 
> I'm not sure if this is just a Java thing or not - @srnagar , could you comment?

By "nested certificate", do you mean a sub-package for certificate client? If yes, then this was discussed in the arch board to have a sub-package for the certificate client.

**Pull request merged**